addons/editor-palette-scroll/addon.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "Custom palette scroll speed",
+  "description": "Change the speed that the palette scrolls.",
+  "tags": ["editor", "codeEditor"],
+  "credits": [
+    {
+      "name": "LilyMakesThings",
+      "link": "https://scratch.mit.edu/users/LilyMakesThings/"
+    }
+  ],
+  "enabledByDefault": false,
+  "userscripts": [
+    {
+      "url": "userscript.js"
```

Comment:
This never runs because it's missing `matches` which is required since not all addons run in  the editor.